getting-started/build.gradle

```diff
@@ -1,16 +1,16 @@
 plugins {
     id 'java'
-    id 'io.quarkus.gradle.plugin' version '0.12.0'
+    id 'io.quarkus'
 }
 
 repositories {
+     mavenLocal()
```

Comment:
This really shouldn't be required? The build is more stable and reproducible without this, and we shouldn't promote needing it IMHO.

Reply:
You can't build the `development` branch without it as we don't publish snapshots.

Reply:
so in an ideal world, this would be yanked by the script doing the releasing, but I guess that's minor